src/main/java/org/folio/edge/sip2/repositories/PatronRepository.java

```diff
@@ -325,11 +324,12 @@ private PatronInformationResponseBuilder addOverdueItems(JsonObject overdues, bo
     });
   }
 
-  private String getPatronPersonalName(Personal personal) {
+  private String getPatronPersonalName(Personal personal, String defaultPersonalName) {
     if (personal != null) {
       return Stream.of(personal.getFirstName(), personal.getMiddleName(), personal.getLastName())
           .filter(Objects::nonNull)
-          .collect(Collectors.joining(" "));
+          .collect(Collectors.collectingAndThen(Collectors.joining(" "),
+              s -> s.isEmpty() ? defaultPersonalName : s));
     }
 
     return "";
```

Comment:
I think we want to return the `defaultPersonalName` here as well so that we have something reasonable to send back to the SC. The SC may have problems with an empty personal name field since it is required.